src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -450,6 +460,74 @@ public void initialLoanDueDateOnCreateWithPrexistingRequests()
 
     assertThat("due date is not the recall due date (2 months)",
         storedLoan.getString("dueDate"), is(expectedDueDate));
+  }
+
+  @Test
+  public void changedDueDateAfterRecallingAnItemShouldRespectTenantTimezone()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final String stockholmTimeZone = "Europe/Stockholm";
+
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    configClient.create(timezoneConfigurationFor(stockholmTimeZone));
+
+    final LoanPolicyBuilder canCirculateRollingPolicy = new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(Period.days(14))
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.days(5))
+      .withClosedLibraryDueDateManagement(KEEP_THE_CURRENT_DUE_DATE.getValue());
+
+    final IndividualResource loanPolicy = loanPoliciesFixture.create(canCirculateRollingPolicy);
+
+    useLoanPolicyAsFallback(loanPolicy.getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.inactiveNotice().getId());
+
+    final DateTime loanDate = DateTime.now(DateTimeZone.UTC).minusDays(3);
+
+    final IndividualResource loan = loansFixture.checkOutByBarcode(
+      smallAngryPlanet, steve, loanDate);
+
+    final String originalDueDate = loan.getJson().getString("dueDate");
+
+    final DateTime requestDate = DateTime.now(DateTimeZone.UTC);
+
+    freezeTime(requestDate);
+
+    requestsFixture.place(
+      new RequestBuilder()
+        .recall()
+        .forItem(smallAngryPlanet)
+        .by(jessica)
+        .withRequestDate(requestDate)
+        .withPickupServicePoint(requestServicePoint));
+
+    final JsonObject storedLoan = loansStorageClient.getById(loan.getId()).getJson();
+
+    assertThat("due date is the original date",
+      storedLoan.getString("dueDate"), not(originalDueDate));
+
+    final DateTime expectedDueDate =
```

Comment:
Seems like this value is equal to the end on loan day, but not to expected due date. Probably `plusDays(5)` should be moved here.

Reply:
Thank you, that is a good spot, I will correct that it in an upcoming pull request